workflows/crisprseq_screening.nf

```diff
@@ -262,6 +262,12 @@ workflow CRISPRSEQ_SCREENING {
     ch_versions = ch_versions.mix(BAGEL2_GRAPH.out.versions)
     }
 
+    if(params.mle_control_sgrna) {
+        ch_mle_control_sgrna = Channel.fromPath(params.mle_control_sgrna)
+    } else {
+        ch_mle_control_sgrna = Channel.empty()
```

Comment:
To make the channel optional, this should be an empty list, not a channel. Otherwise, the process won't run when `ch_mle_control_sgrna` is an empty channel.
```suggestion
        ch_mle_control_sgrna =[]
```